UI/Sources/ShoppingCart/Models/ShoppingCart+Vouchers.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ShoppingCart+Vouchers.swift
+//  Snabble
+//
+//  Created by Uwe Tilemann on 30.12.24.
+//
+
+import SnabbleCore
+import SnabbleAssetProviding
+
+extension VoucherType {
+    var name: String {
+        switch self {
+        case .depositReturn:
+            return Asset.localizedString(forKey: "Snabble.ShoppingCart.DepositReturn.title")
+        default:
+            return ""
+        }
+    }
```

Comment:
Kleinigkeit, aber hier würde ich empfehlen das wir den `unknown` Case hier abfragen, da wir ansonsten bei einem neuen Type keine Fehlermeldung erhalten.

```suggestion
    var name: String {
        switch self {
        case .depositReturn:
            return Asset.localizedString(forKey: "Snabble.ShoppingCart.DepositReturn.title")
        case .unknown:
            return ""
        }
    }
```

Reply:
https://github.com/snabble/snabble-ios-sdk/pull/414/commits/bf4766e0eb70137e6d5a3cad5d0f43ed2a4b7088